components/jaggery-apps/publisher/src/main/publisher/site/themes/wso2/libs/load-tabs.js

```diff
@@ -473,7 +473,7 @@ var loadLC = function () {
                 var initialY = 10;
 // Center the graph
                 inner.attr("transform", "translate(" + [initialX, initialY] + ") scale(" + initialScale + ")");
-                zoom.translate([initialX, initialY]).scale(initialScale).event(svg);
+                zoom.translate([initialX, initialY]).scale(initialScale);
```

Comment:
This is not related to the subscription view. But this was a fix done for another issue.